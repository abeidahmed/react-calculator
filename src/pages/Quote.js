import React from 'react';

const Quote = () => <h1>hello from quote</h1>;

export default Quote;
