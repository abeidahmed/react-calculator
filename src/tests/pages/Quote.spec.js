import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../pages/Quote';

describe('Quote', () => {
  test('renders home page', () => {
    const component = renderer.create(<Quote />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
