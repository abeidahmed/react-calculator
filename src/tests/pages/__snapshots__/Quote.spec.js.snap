// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders home page 1`] = `
<main
  className="sc-bdvvaa PwILa"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
  </p>
</main>
`;
